src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandler.java

```diff
@@ -178,4 +187,16 @@ private void clearTransientFields(SscsCaseData caseData) {
             .unprocessedPostponementRequest(unprocessedPostponementRequest)
             .build());
     }
+
+    private SscsDocument getLatestPostponementDocumentForDwpType(List<SscsDocument> postponementDocuments) {
+        Stream<SscsDocument> filteredStream = postponementDocuments.stream()
+                .filter(f -> DocumentType.POSTPONEMENT_REQUEST.getValue().equals(f.getValue().getDocumentType())
+                        && !isNull(f.getValue().getOriginalPartySender()));
+
+        List<SscsDocument> sortedList = filteredStream.sorted(Comparator.comparing(o -> o.getValue().getDocumentDateAdded()))
```

Comment:
why not just these two operations, there's no other use of the filteredStream variable